mep/accounts/models.py

```diff
@@ -291,20 +292,6 @@ def __str__(self):
         return '%s for account #%s %s' % \
             (self.__class__.__name__, self.account.pk, self.date_range)
 
-    @property
```

Comment:
This is removed because it's now redundant due to the method in `PartialDateMixin` ? Are they consistent in implementation/behavior?

Reply:
This is the case. I think we have the consistency issue with localized dates with months printed vs ISo dates, but I think that's a broader question than this PR.